app/apollo/test/channel.spec.js

```diff
@@ -51,15 +51,46 @@ let presetClusters;
 
 const channel_01_name = 'fake_channel_01';
 const channel_01_uuid = 'fake_ch_01_uuid';
+const channel_01_custom = null;
 
 const channel_02_name = 'fake_channel_02';
 const channel_02_uuid = 'fake_ch_02_uuid';
+const channel_02_custom = [
+  {
+    'key': 'keyA',
+    'val': 'valA',
+  },
+];
 
 const channel_03_name = 'fake_channel_03';
 const channel_03_uuid = 'fake_ch_03_uuid';
+const channel_03_custom = [
+  {
+    'key': 'key_with_null_val',
+    'val': null,
+  },
+  {
+    'key': 'keyB',
+    'val': 'valB',
+  },
+];
```

Comment:
Oh I see, I didn't notice that was an option. Switching to scalar type JSON would definitely clean this up a ton, I'll start switching over to that.

Reply:
Or maybe define new scalar type MAP - see https://www.apollographql.com/docs/apollo-server/schema/custom-scalars/

Reply:
What's the advantage of using a map over the scalar type JSON?

Reply:
Depends on requirements. Maybe you want to ensure users only allowed to pass String values instead of any valid Json. The validations, including duplication of keys, would be done in one place. It's up to you.